apps/remix-ide-e2e/src/tests/compiler_api.test.ts

```diff
@@ -111,11 +119,28 @@ const jsCompileWithOptimization = `(async () => {
       }
       const result = await remix.call('solidity', 'compileWithParameters', contract, params)
       console.log('result ', result)
+  } catch (e) {
+      console.log(e.message)
+  }
+})()`
+
+const jsCompileWithOptimizationDefault = `(async () => {
+  try {
+      const contract = {
+          "storage.sol": {content : \`${simpleContract}\` }
+      }
+      console.log('compile')
+      const params = {
+          optimize: false,
```

Comment:
don't you need to specify compiler version, evm version, language, ... ?

Reply:
I'm changing only this, everything else can go with default

Reply:
ok I don't know if default are handled for these params. Don't they need to be specified?